crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -451,22 +518,25 @@ mod tests {
         // Spill an uncompressed buffer to disk.
         let temp_dir = tempfile::tempdir().unwrap();
         let error_bound = ErrorBound::try_new(0.0).unwrap();
-        let mut buffer = UncompressedInMemoryDataBuffer::new(UNIVARIATE_ID, error_bound);
-        buffer.insert_data(100, 10.0);
+        let mut buffer =
+            UncompressedInMemoryDataBuffer::new(UNIVARIATE_ID, CURRENT_BATCH, error_bound);
+        buffer.insert_data(100, 0, 10.0);
```

Comment:
Is it intentional that the batch index is 100 and the timestamp is 0 now. Previously the timestamp was 100 so it seems the batch index was inserted in the wrong position. Maybe also change it to use the `CURRENT_BATCH` constant.